tests/virtualization/xen/hotplugging.pm

```diff
@@ -35,9 +35,11 @@ sub run_test {
                 record_soft_failure 'bsc#1168124 Bridge network interface hotplugging has to be performed at the beginning.';
                 $self->{test_results}->{$guest}->{"bsc#1168124 Bridge network interface hotplugging has to be performed at the beginning"}->{status} = 'SOFTFAILED';
```

Comment:
This comment is not related to this PR code change. I am just curious about this junit result usage. 

It seems this kind of test_results format is a little uncommon. Generally people define test results like $self->{test_results}->{test target, eg host or guest}->{test case or test step}->{status} =xx, and append detailed error msg like {test_results}->{test target, eg host or guest}->{test case or test step}->{error}=xx, right? But this way seems like {test_results}->{test target, eg host or guest}->{failurexx}->{status} = 'FAILED'.

Is this the only supported way now for the automatic  junit log display in openqa job web ui which you designed earlier? Does the other way given above works?

Reply:
1. That comment is not related to this pull request. It records another problem. It is already self-explanatory, very clear. Please do not try to come up with something that is going to stir things up.
2. The junit test results is by design. If you are not clear, please look back to lib/virt_feature_test_base.pm which provides the description. And also virt_utils module. It has nothing to do with common or uncommon, it suits our usage.
3. I did not invent junit openqa job web ui, I just make it fit with broader range of test status. And try to maintain its simplicity. So currently, this is the only way works.

Reply:
Ok. @Julie-CAO FYI. Your usage in sr-iov is the currently only supported way.